cli/cmd/customer_update.go

```diff
@@ -39,6 +39,11 @@ func (r *runners) InitCustomerUpdateCommand(parent *cobra.Command) *cobra.Comman
 	cmd.Flags().StringVar(&r.args.customerUpdateEmail, "email", "", "Email address of the customer that is to be updated.")
 	cmd.Flags().StringVar(&r.args.customerUpdateType, "type", "dev", "The license type to update. One of: dev|trial|paid|community|test (default: dev)")
 	cmd.Flags().StringVar(&r.outputFormat, "output", "table", "The output format to use. One of: json|table (default: table)")
+
+	cmd.MarkFlagRequired("customer")
+	cmd.MarkFlagRequired("channel")
+	cmd.MarkFlagRequired("name") // until the API supports better patching, this is actually a required field
```

Comment:
Wonder if we shouldn't include name in the example usage since its required (or update the doc string on the flag) ? Nothing the help output indicates its required as-is:

```
➜ bin/replicated customer update --customer=2iTuePdR497n6dDvm17YBCnfsy3 --channel stable --channel beta --default-channel stable
Update available: v0.77.1
To upgrade, run "brew upgrade replicatedhq/replicated/cli"

Usage:
  replicated customer update --customer <id> [options] [flags]

Flags:
      --airgap                      If set, the license will allow airgap installs.
      --channel stringArray         Release channel to which the customer should be assigned (can be specified multiple times)
      --custom-id string            Set a custom customer ID to more easily tie this customer record to your external data systems
      --customer string             The ID of the customer to update
      --default-channel string      Which of the specified channels should be the default channel. if not set, the first channel specified will be the default channel.
      --email string                Email address of the customer that is to be updated.
      --embedded-cluster-download   If set, the license will allow embedded cluster downloads.
      --ensure-channel              If set, channel will be created if it does not exist.
      --expires-in duration         If set, an expiration date will be set on the license. Supports Go durations like '72h' or '3600m'
      --geo-axis                    If set, the license will allow Geo Axis usage.
      --gitops                      If set, the license will allow the GitOps usage.
      --helmvm-cluster-download     If set, the license will allow helmvm cluster downloads.
  -h, --help                        help for update
      --identity-service            If set, the license will allow Identity Service usage.
      --kots-install                If set, the license will allow KOTS install. Otherwise license will allow Helm CLI installs only. (default true)
      --name string                 Name of the customer
      --output string               The output format to use. One of: json|table (default: table) (default "table")
      --snapshot                    If set, the license will allow Snapshots.
      --support-bundle-upload       If set, the license will allow uploading support bundles.
      --type string                 The license type to update. One of: dev|trial|paid|community|test (default: dev) (default "dev")

Global Flags:
      --app string     The app slug or app id to use in all calls
      --token string   The API token to use to access your app in the Vendor API

➜ echo $?
1
```